src/main/scala/dcos/metronome/JobApplicationLoader.scala

```diff
@@ -60,23 +59,26 @@ class JobComponents(context: Context) extends BuiltInComponentsFromContext(conte
     jobsModule.pluginManger,
     httpErrorHandler,
     assets,
-    jobsModule.queueModule.launchQueueService)
+    jobsModule.queueModule.launchQueueService,
+    jobsModule.actorsModule)
 
   def schedulerService = jobsModule.schedulerModule.schedulerService
 
   lazy val wsClient: WSClient = {
-    val parser = new WSConfigParser(configuration, environment)
+    val parser = new WSConfigParser(configuration.underlying, environment.classLoader)
     val config = new AhcWSClientConfig(wsClientConfig = parser.parse())
     val builder = new AhcConfigBuilder(config)
-    val logging = new AsyncHttpClientConfig.AdditionalChannelInitializer() {
-      override def initChannel(channel: io.netty.channel.Channel): Unit = {
-        channel.pipeline.addFirst("log", new io.netty.handler.logging.LoggingHandler(classOf[WSClient]))
-      }
-    }
+    // org.asynchttpclient.AsyncHttpClientConfig.AdditionalChannelInitializer
+    //    val logging = new AsyncHttpClientConfig.AdditionalChannelInitializer() {
+    //      override def initChannel(channel: play.shaded.ahc.io.netty.channel.Channel): Unit = {
+    //        channel.pipeline.addFirst("log", new io.netty.handler.logging.LoggingHandler(classOf[WSClient]))
+    //      }
+    //    }
     val ahcBuilder = builder.configure()
-    ahcBuilder.setHttpAdditionalChannelInitializer(logging)
+    // play.shaded.ahc.org.asynchttpclient.AsyncHttpClientConfig.AdditionalChannelInitializer
+    //    ahcBuilder.setHttpAdditionalChannelInitializer(logging)
     val ahcConfig = ahcBuilder.build()
-    new AhcWSClient(ahcConfig)
+    new AhcWSClient(StandaloneAhcWSClient())
```

Comment:
We still have to configure the logging.

Reply:
yeah, screw that for now :) hopefully we won't need to bump play in the future